File: CS/T300213wf/App_Code/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

public class Product
{
	public int Id { get; set; }
	public string Name { get; set; }
	public List<Store> Stores { get; set; }

	public Product() {
		Stores = new List<Store>();
	}
}

File: VB/T300213wf/App_Code/Product.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Public Class Product
    Public Property Id() As Integer
    Public Property Name() As String
    Public Property Stores() As List(Of Store)

    Public Sub New()
        Stores = New List(Of Store)()
    End Sub
End Class
